src/compressed_tensors/compressors/model_compressors/model_compressor.py

```diff
@@ -254,6 +258,92 @@ def __init__(
                 quantization_config.format, config=quantization_config
             )
 
+    def get_missing_module_keys(self, model: Module) -> List[str]:
+        """
+        Identifies the expected missing keys in the compressed state_dict
+        based on the model structure and applied compression techniques.
```

Comment:
can you leave a comment that a missing key is related to "weight' not being in the checkpoint, as a result of the compressed format?